datafusion/expr/src/udaf.rs

```diff
@@ -818,6 +826,26 @@ pub mod aggregate_doc_sections {
     };
 }
 
+/// Status of an Aggregate Expression's Set Monotonicity
+#[derive(Debug, Clone)]
+pub enum AggregateExprSetMonotonicity {
+    /// Ordering exists as ascending
```

Comment:
I think it would help to explain what ascending means here 

Something like: 
```suggestion
    /// Ordering exists as ascending: value of the aggregate always increases
    /// (or stay the same) as new values are seen. 
```

Reply:
I think we should probably use the naming `AggregateExprSetMonotonicity::Increasing`, which would be self-documenting. I will make the change.